src/models/CushyScriptL.ts

```diff
@@ -109,12 +109,16 @@ export class CushyScriptL {
         const codeJS = this.data.code
         const APPS: App<WidgetDict>[] = []
 
+        let appIndex = 0
         // 1. setup DI registering mechanism
-        const registerAppFn = (a1: string, a2: App<any>): void => {
+        const registerAppFn = (a1: string, a2: App<any>): AppRef<any> => {
             const app: App<WidgetDict> = typeof a1 !== 'string' ? a1 : a2
             const name = app.metadata?.name ?? basename(this.relPath)
             console.info(`[💙] found action: "${name}"`, { path: this.relPath })
             APPS.push(app)
+            const appID = asCushyAppID(this.relPath + ':' + appIndex++) // 🔴 SUPER UNSAFE
+            // console.log(`[👙] >> appID==`, appID)
+            return { $DO_NOT_USE: 0 as any, id: appID }
```

Comment:
Why not just use `$Output`? I think the `$` is sufficient to let people know to leave it alone